tests/app/forms/handlers/test_date_handler.py

```diff
@@ -0,0 +1,300 @@
+# pylint: disable=unused-argument
```

Comment:
Some of these tests should be moved into test files for the different date handlers. Other tests should be moved into test files for the date fields.